ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -52,14 +73,14 @@ sub process_path {
     # Pull out the values and check that they really are defined.
     my $path = $pathentry->{path};
     if (!defined($path)) {
-        $self->error("entry with undefined path");
+        $self->error("Entry with undefined path");
```

Comment:
add the undefined `$path` to the message? (it looks very useless without it imho).